Exception/LifecircleException.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/*
+ * This file is part of the ecentria group, inc. software.
+ *
+ * (c) 2016, ecentria group, inc.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ecentria\Libraries\EcentriaAPIEventsBundle\Exception;
+
+/**
+ * Abstract Flow Exception
+ * 
+ * @copyright   2016 OpticsPlanet, Inc
+ * @author      Eugene Boiarynov <ievgen.boiarynov@opticsplanet.com>
+ */
+abstract class LifecircleException extends \Exception
```

Comment:
It's Lifecycle, not Lifecircle.

Can you explain why you called the base class "LifecycleException"? I think ConsumerException would be a better name.